ddraw/IDirect3DDeviceX.cpp

```diff
@@ -1848,7 +1931,21 @@ HRESULT m_IDirect3DDeviceX::LightEnable(DWORD dwLightIndex, BOOL bEnable)
 			return DDERR_GENERIC;
 		}
 
-		return (*d3d9Device)->LightEnable(dwLightIndex, bEnable);
+		hr = (*d3d9Device)->LightEnable(dwLightIndex, bEnable);
```

Comment:
`hr` could have been declared here instead of above

Reply:
Good call.  Will fix this in the next update.